pyproject.toml

```
@@ -32,7 +32,7 @@ requires-python = ">=3.8"
 dependencies = [
     "numpy>=1.22.0",
     "matplotlib>=3.2.1",
-    "scipy>=1.4.1",
+    "scipy>=1.10.0",
```

```suggestion
    "scipy",
```

I would prefer to just leave this unconstrained. Or did someone ever test it does not work prior to 1.4.1?

---

```suggestion
    "scipy>=1.4.1",
```

Then lets revert to this :+1: